.vscode/settings.json

```diff
@@ -7,5 +7,8 @@
   "eslint.format.enable": true,
   "[typescriptreact]": {
     "editor.defaultFormatter": "dbaeumer.vscode-eslint"
+  },
+  "files.exclude": {
+    "**/*.scss.ts": true
```

Comment:
I've added this in combination with the [_Make Hidden_ plugin](https://marketplace.visualstudio.com/items?itemName=devine-davies.make-hidden) so you don't see them in the in Explorer tab in VS Code, since you don't need to edit those files any way. I guess if you don't have the plugin this doesn't matter, but I found it really nice